bundle/manifests/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -82,7 +90,7 @@ spec:
                 description: Deploy the NFD-Topology-Updater NFD-Topology-Updater
                   is a daemon responsible for examining allocated resources on a worker
                   node to account for resources available to be allocated to new pod
-                  on a per-zone basis https://kubernetes-sigs.github.io/node-feature-discovery/v0.10/get-started/introduction.html#nfd-topology-updater
+                  on a per-zone basis https://kubernetes-sigs.github.io/node-feature-discovery/v0.12/get-started/introduction.html#nfd-topology-updater
```

Comment:
Should we use `master` here, or `stable`? It kinda would make sense that the operand master branch try to be compatible with the master branch of the operand. Then on release branch we'd update these. WDYT?